python/rikai/spark/functions/vision.py

```diff
@@ -106,6 +108,7 @@ def numpy_to_image(array: ndarray, uri: str) -> Image:
 def video_to_images(
     video,
     segment: Segment = Segment(0, -1),
+    youtube_uri: str = tempfile.gettempdir(),
```

Comment:
this is actually a python gotcha: https://8thlight.com/blog/shibani-mookerjee/2019/05/07/some-common-gotchas-in-python.html#:~:text=Python's%20default%20arguments%20are%20evaluated,to%20the%20function%20as%20well.

make the default None and then create a tempdir inside the function if it's none

Reply:
made the default directory an empty string so we could keep the typing